lib/eventemitter2.js

```diff
@@ -40,18 +40,25 @@
 
   function logPossibleMemoryLeak(count, eventName) {
     var errorMsg = '(node) warning: possible EventEmitter memory ' +
-        'leak detected. %d listeners added. ' +
+        'leak detected. ' + count + ' listeners added. ' +
         'Use emitter.setMaxListeners() to increase limit.';
 
     if(this.verboseMemoryLeak){
-      errorMsg += ' Event name: %s.';
-      console.error(errorMsg, count, eventName);
-    } else {
-      console.error(errorMsg, count);
+      errorMsg += ' Event name: ' + eventName + '.';
     }
 
-    if (console.trace){
-      console.trace();
+    if(process && process.emitWarning){
```

Comment:
emitWarning doesn't need to have a typeof check, just checking if it's falsy is enough since it isn't a global. If you're in strict mode and attempt to access a variable that doesn't exist, it'll throw an error when parsing, using `typeof` prevents the error from being thrown.